packages/core/admin/ee/server/services/review-workflows/review-workflows.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const { WORKFLOW_MODEL_UID, STAGE_MODEL_UID } = require('../../constants/workflows');
+
+module.exports = ({ strapi }) => ({
+  async bootstrap() {
+    const wfCount = await strapi.entityService.count(WORKFLOW_MODEL_UID);
+    const stagesCount = await strapi.entityService.count(STAGE_MODEL_UID);
+
+    if (wfCount + stagesCount === 0) {
```

Comment:
why looking at the stages count? can we have 0 workflows and more than 0 stages?

Reply:
If there are any stages or workflows, we already have our default workflow, or the user has tricked his database.